core/app/models/spree/wallet/add_default_payment.rb

```diff
@@ -0,0 +1,24 @@
+class Spree::Wallet::AddDefaultPayment
```

Comment:
We should consider to call this class `DefaultPaymentBuilder` as we actually don't add something, we build something. Otherwise great!